### PR TITLE
chore: slack client, csv service and project model changes

### PR DESCRIPTION
**Pull request by IrakliJani**
### Description:

changes are necessary for bot

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/models/ProjectModel/ProjectModel.ts [hunk 1]
```diff
@@ -752,41 +750,42 @@ export class ProjectModel {
...
-                                for (
-                                    let i = 0;
-                                    i < catalogItems.length;
-                                    i += chunkSize
-                                ) {
-                                    const chunk = catalogItems
-                                        .slice(i, i + chunkSize)
-                                        .map((catalogItem) => ({
+                                const inserts = await this.database
+                                    .batchInsert(
```
Comment (rephus) [inline packages/backend/src/models/ProjectModel/ProjectModel.ts:764]: nice, I didn't know about this utility function. 

> By default, the chunkSize is set to 1000.

So in theory it should be the same.

**Review by almeidabbm — APPROVED** [review]

**Review by rephus — APPROVED** [review]
Cool, LGTM if tests are passing (and catalog is searchable) then we can merge. 

Are you planning on using the list of inserted catalog items somewhere ?

File: packages/backend/src/models/ProjectModel/ProjectModel.ts (hunk 1 above)
Comment (IrakliJani) [reply packages/backend/src/models/ProjectModel/ProjectModel.ts:764]: @rephus I'll DM the details

**Comment by rephus** [comment]
Tested on catalog, it works !

**Pull request merged**